app/main.py

```diff
@@ -2,0 +1,24 @@
+import os
+
+
+def move_file(command: str) -> None:
+    split_command = command.split()
+    mv, file_1, path_file_2 = split_command
+
+    if "/" in command:
+        path = path_file_2[0:path_file_2.rfind("/")]
+
+        if not os.path.exists(path):
+            os.makedirs(path)
+        with (open(file_1, "r") as file_in,
+              open(path_file_2, "w") as file_out):
```

Comment:
Code Style: To concatenate parts of the path, it's better to use 'os.path.join'. This will make your app cross-platform and it will use either '/' or '\' depending on the OS.